portfolio/src/main/webapp/js/comments.js

```diff
@@ -50,3 +71,42 @@ function addCommentToPage(comment) {
   newComment.appendChild(authorFooter);
   commentContainer.appendChild(newComment);
 }
+
+/**
+ * Attempts to post a comment using the comment form data.
+ * @param {SubmitEvent} event
```

Comment:
Is the param a `SubmitEvent`? I don't see that particular type here https://developer.mozilla.org/en-US/docs/Web/API/Event, maybe we just call it `Event`? That should cover it.

Reply:
Hm. I saw it somewhere but yeah the resource you sent is clear. Will change.